recipes/biapy/meta.yaml

```diff
@@ -0,0 +1,68 @@
+{% set name = "biapy" %}
+{% set version = "3.6.2" %}
+{% set python_min = "3.10" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/BiaPyX/BiaPy/archive/refs/tags/v{{ version }}.tar.gz
```

Comment:
Any reason not to get this from pypi? That tarball is a couple of hundred kb, but from github is almost 75 MB